cli/lib/utils.ts

```diff
@@ -52,18 +52,18 @@ const AUTOGENERATED_COMMENT = `
   ${padEnd('//', 80, '/')}
 `
 
-export type ScriptType = 'rules' | 'db'
+export type CodeType = 'rules' | 'db' | 'login'
 
-/** Generate the code of the Auth0 script as a string */
-export async function generateScript(
+/** Generate the Auth0 code as a string */
```

Comment:
Hmm, I think the previous comment was a bit clearer – it's not clear to me what 'the Auth0 code' is on a quick scan, it sounds like it could be e.g. a 2FA code, or the code from an Auth0 library.

Perhaps something like this is less ambiguous
> Generate the code of an Auth0 script from a rule definition